src/components/Admin/RobotTeamAssigner.tsx

```diff
@@ -59,7 +60,7 @@ const RobotTeam = ({
 
 	return (
 		<>
-			<label htmlFor={`robot-${address}-name`}>Team</label>
+			<label htmlFor={`robot-${address}-name`}>Team {':'}</label>
```

Comment:
```suggestion
			<label htmlFor={`robot-${address}-name`}>Team:</label>
```

Brackets not needed.